core/modules/system/system.theme.inc

```diff
@@ -196,7 +195,12 @@ function theme_system_modules_fieldset($variables) {
 
     // Hide modules that can't be disabled at all.
     if (!$module["#required_by_distribution"]) {
-      $rows[] = $row;
+      $classes = array();
+      if (isset($module['#source'])) {
+        $classes[] = $module['#source'];
+      }
```

Comment:
More "source" stuff that should probably be removed?

Reply:
fixed